tests/php/i18n/i18nTextCollectorTest.php

```diff
@@ -71,6 +71,10 @@ public function testConcatenationInEntityValues()
         );
     }
 
+    /**
+     * @expectedException PHPUnit_Framework_Error_Notice
+     * @expectedExceptionMessage Missing localisation default for key i18nTestModule.INJECTIONS_3
```

Comment:
Since this test mixes both asserts and expected exceptions, I suggest replacing this with `$this->expectException() / $this->expectExceptionMessage()` rather than annotations.